e2e-tests/oauth-flow/openid4vp/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.7"
+services:
+  nodeA-backend:
```

Comment:
why do we need NGINX in front, we can terminate TLS on the Nuts node in this case, I think?

As the Nuts network isn't needed.